extension/src/background/store.ts

```diff
@@ -2,9 +2,60 @@ import { combineReducers } from "redux";
 import { configureStore } from "@reduxjs/toolkit";
 
 import { sessionSlice } from "background/ducks/session";
+import {
+  SESSION_STORAGE_ENABLED,
+  dataStorageAccess,
+  sessionStorage,
+  browserStorage,
+} from "./helpers/dataStorage";
 
-export const store = configureStore({
+// Session storage can be used to persist redux store in Manifest v3 because service workers go idle after 30 seconds
+// Session storage is not currently supported in Firefox, which blocks our migration to using this by default
+const REDUX_STORE_KEY = "redux-store";
```

Comment:
this is pretty NIT: but would you mind calling this something a bit more descriptive than `redux-store`? Maybe something that conveys that it's the user's session creds or something 🤔

Reply:
ah yeah fair, how about `session-store`?